pkg/api/http_server.go

```diff
@@ -309,10 +309,6 @@ func (hs *HTTPServer) configureHttp2() error {
 func (hs *HTTPServer) newMacaron() *macaron.Macaron {
 	macaron.Env = hs.Cfg.Env
 	m := macaron.New()
-
-	// automatically set HEAD for every GET
-	m.SetAutoHead(true)
```

Comment:
Is this happening automatically in some other place I cannot see?

Reply:
Yes, https://github.com/grafana/grafana/pull/37795/files#diff-a931a8a60996a6562f6346407b0f4345ebbdcf640a142799a4fa72170d6b14e8R172
Previously there was a flag, whether we want to register GET and HEAD handlers at the same time, but this flag was always hard-coded to `true`, so instead I modified the Get() method to always call Head().

Reply:
Okay great